.all-contributorsrc

```diff
@@ -1,52 +0,0 @@
-{
```

Comment:
Oops. We don't want to delete this file altogether. Just restore its contents to how they are in the master branch 🙂